src/components/Button/button.component.jsx

```diff
@@ -1,23 +1,23 @@
 import propTypes from 'prop-types';
 import './button.styles.css';
 
-const handleClick = e => {
-  e.target.classList.add('sel');
-};
+const handleClick = props => props.clickHandler;
 
 const Button = props => {
-  const { name } = props;
+  const { name, title } = props;
   return (
-    <button type="button" onClick={handleClick} className="calculator-key">{ name }</button>
+    <button type="button" onClick={handleClick(props)} className="calculator-key" title={title}>{ name }</button>
   );
 };
 
 Button.propTypes = {
   name: propTypes.string,
+  title: propTypes.string,
```

Comment:
**[Required]**
Let us validate handleClick as well. All props have to be validated.